Project.toml

```diff
@@ -19,6 +20,7 @@ Git = "1.2.1"
 GitHub = "5.4"
 JSON3 = "1.5.1"
 LicenseCheck = "0.2"
+RegistryInstances = "0.1"
 julia = "1.6"
```

Comment:
I'd be ok to go with v1.7 only if that made things easier, but I presume `RegistryInstances` should take care of cross-version compatibility, right?

Reply:
Yeah, works on 1.6+: https://github.com/GunnarFarneback/RegistryInstances.jl#about-this-package